E2E-tests/config/substrate/devnet_nodes.json

```diff
@@ -157,6 +157,11 @@
                 "lock": 817414141,
                 "send": 298945143
             }
-        }
+        },
+        "permissioned_min_ariadne_d_param_value": 1,
+        "permissioned_max_ariadne_d_param_value": 5,
+        "trustless_min_ariadne_d_param_value": 1,
+        "trustless_max_ariadne_d_param_value": 5,
```

Comment:
test fails if you set min==max
```
        "permissioned_min_ariadne_d_param_value": 4,
        "permissioned_max_ariadne_d_param_value": 4,
        "trustless_min_ariadne_d_param_value": 1,
        "trustless_max_ariadne_d_param_value": 1,
```
```
________________ TestCommitteeDistribution.test_update_d_param _________________
tests/committee/test_committee.py:65: in test_update_d_param
    new_p = np.random.randint(p_floor, p_ceil)
numpy/random/mtrand.pyx:782: in numpy.random.mtrand.RandomState.randint
    ???
numpy/random/_bounded_integers.pyx:1334: in numpy.random._bounded_integers._rand_int64
    ???
E   ValueError: low >= high
```

Reply:
fixed - I put a guard around this condition.